mantidproject/mantid
# Add python editor to workbench

Description of work.

**To test:**

* Start the workbench
* The editor should appear as one of the main widgets. Try it out.

Fixes #21251 

Please note that the lack of unit test coverage in some areas will be addressed in #21563. This is not yet going out to production but is required for the upcoming user meeting.

**Release Notes** 
*Does not need to be in the release notes.*

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
